test-requirements.txt

```diff
@@ -0,0 +1 @@
+ansible-lint==2.3.6
```

Comment:
This PR is confusing to me. What problem is being solved here? A more
descriptive PR text would be helpful. Travis is currently building
successfully on master.


On 02/16, Laurent Arnoud wrote:
> > @@ -0,0 +1 @@
> > +ansible-lint==2.3.6
> 
> Only ansible-lint `2.3.6` is working after there will be sudo/become errors that are not merged yet.
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/sovereign/sovereign/pull/487/files#r53078021